k2/csrc/context.cu

```diff
@@ -77,7 +77,7 @@ void ParallelRunnerActive::Finish() {
     for (std::size_t i = 0; i != streams_.size(); ++i) {
       // create and record event on `stream_[i]`, and wait on c_->GetCudaStream
       cudaEvent_t event;
-      auto ret = cudaEventCreate(&event);
+      auto ret = cudaEventCreateWithFlags(&event, cudaEventDisableTiming);
```

Comment:
can we have a comment to explain why we are using this flag?

Reply:
From https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__EVENT.html#group__CUDART__EVENT_1g7b317e07ff385d85aa656204b971a042

> cudaEventDisableTiming: Specifies that the created event does not need to record timing data. Events created with this flag specified and the cudaEventBlockingSync flag not specified will provide the **best performance** when used with cudaStreamWaitEvent() and cudaEventQuery().

I think the documentation of `cudaEventCreateWithFlags` explains the flag clearly.